csrc/scheduler/normalization.cpp

```diff
@@ -1777,6 +1777,7 @@ void schedulePersistentKernelInnerOuter(
   // otherwise the parallelization will be propagated to the first domain of tvs
   // in boundaryNodesSet, because that domain was calculated from the partial
   // reduction and these tensors are not scheduled yet.
+  auto selected_tvs_inner = ir_utils::allTvsExcept(fusion, boundaryNodesSet);
```

Comment:
Doesn't this set include tensors used for outer reductions? 

Also, shouldn't the comment above be updated as well?

Reply:
Propagation from inner reduction tvs to outer reduction can't happen since the outer reduction part is not scheduled yet. what happedn in that issue is the propagation from outer to inner where the inner reduction part is already scheduled.

Reply:
I'm talking just about this statement.

Reply:
I changed this part to make it same with outer reduction parallelization propagation.